### ISSUE TITLE
Create class for family income

### ISSUE THREAD

**Issue by aellenhicks** [ufbmi/OMRSE#71]
Definition proposed by zhengj2007 in issue #62 

family income: no definition yet (parent: IAO_0000030 information content entity)

**Comment by aellenhicks** [comment]
Does this refer to the income of a family that lives together?

**Comment by cstoeckert** [comment]
Yes this refers to a household (people living together). Could label / define as that instead of family if it helps clarify.

**Comment by dexink** [comment]
@hoganwr does this still need added? (currently, no term for 'household income' or 'family income' seems to exist)

**Comment by hoganwr** [comment]
Note: this is a NTR. We also discussed "household income", which has a class already in BCIO. BCIO has as parent class "object aggregate" but OMRSE doesn't define income as an object, so that wouldn't cohere well.

It is likely that family income would be a subclass of household income, where the members of the household have familial relationships. 

Note that we currently have "monetary personal income data item" as proposed parent class, which enables being about one or more persons.

**Comment by dillerm** [comment]
Proposals for family and household income:

- **‘household income’** = “A monetary personal income data item that is about a household.”
- **‘family income’** = “Household income that pertains to members of a household related by birth, marriage, or adoption.”
